src/main/java/mate/academy/spring/security/PasswordMatchesValidator.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.security;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import mate.academy.spring.annotations.PasswordMatches;
+import mate.academy.spring.dto.UserDto;
+
+public class PasswordMatchesValidator implements ConstraintValidator<PasswordMatches, Object> {
```

Comment:
```suggestion
public class PasswordMatchesValidator implements ConstraintValidator<PasswordMatches, UserDto> {
```

@AndreyDMA Can we do this in this way? Also, can we cover this with JUnit tests in the future?

Reply:
I can't do it in this way, becouse I can't change signature of interface...

Reply:
@AndreyDMA 
https://github.com/TatianaPirch/jv-spring-hibernate/blob/master/src/main/java/mate/academy/spring/security/PasswordMatchesValidator.java#L10

Reply:
please fix and merge

Reply:
if you have a questions - ping me in slack